packages/components/src/Tooltip/src/TooltipTrigger.tsx

```diff
@@ -0,0 +1,55 @@
+import { TooltipTrigger as RACTooltipTrigger, type TooltipProps as RACTooltipProps, type TooltipTriggerComponentProps as RACTooltipTriggerProps } from "react-aria-components";
+
+import { TooltipTriggerContext } from "./TooltipTriggerContext.ts";
+
+export type TooltipPlacement = "start" | "end" | "right" | "left" | "top" | "bottom";
+
+type PropsToPick = "shouldFlip" | "containerPadding" | "offset" | "crossOffset" | "isOpen" | "defaultOpen" | "onOpenChange";
+
+export interface TooltipTriggerProps extends RACTooltipTriggerProps, Pick<RACTooltipProps, PropsToPick> {
+    /**
+   * The placement of the element with respect to its anchor element.
+   *
+   * @default 'top'
+   */
+    placement?: TooltipPlacement;
+}
+
+/**
+ * A TooltipTrigger wraps a trigger element and Tooltip, handling visibility and positioning.
+ *
+ * [View Documentation](TODO)
+ */
+export function TooltipTrigger(props: TooltipTriggerProps) {
+    const {
+        children,
+        containerPadding = 16, /* Should this be on the trigger or the actual tooltip component? */
```

Comment:
comment here. Did you figure out the answer?

Reply:
no, this was a question. personally I would put it on Tooltip and not the trigger since it's spacing surrounding the tooltip popup. What do you think?

Reply:
Since it's related to positionning and link to the trigger, i'd keep it here